api-reference/connect/desktop/web.mdx

```diff
@@ -1,241 +1,406 @@
+
+Below is a code snippet showcasing a structured configuration of `ConnectKitProvider`, leveraging the parameters listed above.
+
+```tsx connectkit.tsx
+'use client';
+
+import { ConnectKitProvider, createConfig } from '@particle-network/connectkit';
+import { authWalletConnectors } from '@particle-network/connectkit/auth';
+import { mainnet, solana } from '@particle-network/connectkit/chains';
+import { evmWalletConnectors } from '@particle-network/connectkit/evm';
+import { injected as solaInjected, solanaWalletConnectors } from '@particle-network/connectkit/solana';
+import { wallet, EntryPosition } from '@particle-network/connectkit/wallet';
+import React from 'react';
+
+const config = createConfig({
+    projectId: 'replace it with particle project id', // replace this.
+    clientKey: 'replace it with particle client key', // replace this.
+    appId: 'replace it with particle app id', // replace this.
+    appearance: {
+        // optional, define wallet label and sort wallets.
+        recommendedWallets: [
+            { walletId: 'metaMask', label: 'Recommended' },
+            { walletId: 'coinbaseWallet', label: 'popular' },
+        ],
+        splitEmailAndPhone: false,
+        collapseWalletList: false,
+        hideContinueButton: false,
+        //  optional, sort wallet connectors
+        connectorsOrder: ['email', 'phone', 'social', 'wallet'],
+        language: 'en-US',
+        mode: 'light', // dark or auto.
+        theme: {
+            '--pcm-accent-color': '#ff4d4f',
+            // ... other options
+        },
+        logo: 'https://...',
+        filterCountryCallingCode: (countrys) => {
+            // set only support USA, default support all country code.
+            return countrys.filter((item) => item === 'US');
+        },
+    },
+    walletConnectors: [
+        evmWalletConnectors({
+            metadata: { name: 'My App', icon: '', description: '', url: '' }, // optional, your app metadata, use for WalletConnect and Coinbase.
+            walletConnectProjectId: 'replace it with WalletConnect project id', // optional, WalletConnect project id.
+        }), // optional, you need to configure it when using social or email/phone connect.
+        authWalletConnectors(), // optional, you need to configure it when using EVM chains.
+        solanaWalletConnectors(), // optional, you need to configure it when using solana.
+    ],
+    plugins: [
+        wallet({
+            entryPosition: EntryPosition.BR,
+            visible: true,
+        }), // optional, embedded wallet, support solana and evm chains.
+    ],
+    chains: [mainnet, solana],
+});
+
+// Wrap your application with this component.
+export const ParticleConnectkit = ({ children }: React.PropsWithChildren) => {
+    return <ConnectKitProvider config={config}>{children}</ConnectKitProvider>;
+};
 ```
 
+You can save this code snippet to a file named `connectkit.tsx`, and use `ParticleConnectkit` wrap your `App`.
+
```

Comment:
We say to use this as an independent component, but we discuss the index file in the previous section. 

In this format is not very clear that we suggest to keep `ConnectKitProvider` in its own component and then import it in your index file to wrap the app with it 👇

```tsx
// Wrap your application with this component.
export const ParticleConnectkit = ({ children }: React.PropsWithChildren) => {
    return <ConnectKitProvider config={config}>{children}</ConnectKitProvider>;
};
```